README.rst

```diff
@@ -44,17 +44,11 @@ Start new build:
     import asyncio
     import aiojenkins
 
-    jenkins = aiojenkins.Jenkins('http://your_server/jenkins', 'user', 'password')
-
     async def example():
-        await jenkins.builds.start('job_name', dict(parameter='test'))
-
-    loop = asyncio.get_event_loop()
-    try:
-        loop.run_until_complete(example())
-    finally:
-        loop.run_until_complete(jenkins.close())
-        loop.close()
+        async with aiojenkins.Jenkins('http://your_server/jenkins', 'user', 'password') as jenkins:
+            await jenkins.builds.start('job_name', dict(parameter='test'))
+
+    asyncio.run(example())
```

Comment:
Available as of python3.7, which this project lists as its minimum version
https://docs.python.org/3/library/asyncio-runner.html#running-an-asyncio-program

Reply:
Okay, but please add example with context manager as additional to just simple definition

Reply:
Sounds good, will push that shortly

Reply:
Done